src/caveexpress/shared/network/messages/TargetCaveMessage.h

```diff
@@ -7,27 +7,35 @@ namespace caveexpress {
 class TargetCaveMessage: public IProtocolMessage {
 private:
 	uint8_t _targetCave;  // 0 none,  100 stone
+	float _angle;  // arrow angle to target cave  -1 if none
```

Comment:
there is no need to send this as a 4 byte float value - uint16_t would be enough here.